ibm/data_source_ibm_pi_instance.go

```diff
@@ -173,6 +177,9 @@ func dataSourceIBMPIInstancesRead(d *schema.ResourceData, meta interface{}) erro
 	d.Set("storage_type", powervmdata.StorageType)
 	d.Set("storage_pool", powervmdata.StoragePool)
 
+	if &powervmdata.LicenseRepositoryCapacity != nil {
```

Comment:
You are checking address of a property. I don't think there is a need of the check as it is int64.
```
the address of a variable cannot be nil (SA4022)go-staticcheck
```